bitcoin-extensions/bitcoin.html

```diff
@@ -0,0 +1,33 @@
+<html>
+	<head>
+		<title>Bitcoin Price Index Watcher</title>
+		<link href="styles.css" rel="stylesheet" type="text/css">
+	</head>
+	<body>
+		<img id="logo" src="bitcoin.png" />
+		<div id="data"></div>
```

Comment:
You might want to wrap the data with some another div, and maybe add a container class to it. That'll make it easier to style things and space them out nicely.

Reply:
Also you could have multiple div inside the container div, with id's like `data1`, `data2`, etc. (or better names)

Then down below...